odoo/debugger/tests.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import re
+import sys
+
+from psycopg2.extensions import STATUS_READY
+
+from odoo.api import Environment
+from odoo.modules.registry import Registry
+from odoo.service.server import server
+from odoo.tools import config
+
+_logger = logging.getLogger(__name__)
+__all__ = ['run_tests']
+
+TEST_MODULE_NAME_PATTERN = re.compile(r'^odoo\.addons\.\w+\.tests')
+
+
+def run_tests(env: Environment, modules: list[str], test_tags: str):
```

Comment:
This could be in `odoo.tests`.

Reply:
I think that it is not an issue. The comment says to "avoid" importing it. You could tweak the condition to disable it or leave as is.

Reply:
As for the
```
_logger.error(
        "Importing test framework"
        ", avoid importing from business modules and when not running in test mode",
        stack_info=True,
    )
```

now it also logged for standalone tests.
Do you think it make sense to add a condition `if threading.current_thread() is threading.main_thread()`
or we add a `config['command']` and log it only `config['command'] == 'server'`